eng/pipelines/templates/jobs/vmr-synchronization.yml

```diff
@@ -25,6 +25,8 @@ jobs:
   - template: /eng/common/templates/variables/pool-providers.yml
   - name: vmrPath
     value: $(Agent.BuildDirectory)/vmr
+  - name: vmrBranchName
+    value: $[replace(parameters.vmrBranch, 'refs/heads/', '')]
```

Comment:
I think this can use the `${{ }} form, since these are template parameters and do not need to be evaluated at runtime.

Reply:
Yeah, that should work